controllers/rhmi/bootstrapReconciler.go

```diff
@@ -643,9 +652,9 @@ func (r *Reconciler) reconcilerGithubOauthSecret(ctx context.Context, serverClie
 
 func generateSecret(length int) string {
 	chars := []rune("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789")
-	rnd, _ := rand.Int(rand.Reader, big.NewInt(int64(len(chars))))
 	buf := make([]rune, length)
 	for i := range buf {
+		rnd, _ := rand.Int(rand.Reader, big.NewInt(int64(len(chars))))
```

Comment:
```suggestion
		rnd, err := rand.Int(rand.Reader, big.NewInt(int64(len(chars))))
```
Could we handle this error instead of ignoring it ? 🤔

Reply:
done